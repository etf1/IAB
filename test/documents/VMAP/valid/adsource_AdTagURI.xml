<!-- AdSources with different templateType values -->
<vmap:VMAP version='1.0' xmlns:vmap='http://www.iab.net/vmap-1.0'>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start" >
        <vmap:AdSource allowMultipleAds='false' followRedirects='true' id='1' >
            <vmap:AdTagURI templateType="vast1">
                <![CDATA[http://7b9df.v.fwmrm.net/ad/1]]>
            </vmap:AdTagURI>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start" >
        <vmap:AdSource allowMultipleAds='false' followRedirects='true' id='1' >
            <vmap:AdTagURI templateType="vast2">
                <![CDATA[http://7b9df.v.fwmrm.net/ad/1]]>
            </vmap:AdTagURI>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start" >
        <vmap:AdSource allowMultipleAds='false' followRedirects='true' id='1' >
            <vmap:AdTagURI templateType="vast3">
                <![CDATA[http://7b9df.v.fwmrm.net/ad/1]]>
            </vmap:AdTagURI>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start" >
        <vmap:AdSource allowMultipleAds='false' followRedirects='true' id='1' >
            <vmap:AdTagURI templateType="proprietary">
                <![CDATA[http://7b9df.v.fwmrm.net/ad/1]]>
            </vmap:AdTagURI>
        </vmap:AdSource>
    </vmap:AdBreak>
</vmap:VMAP>
