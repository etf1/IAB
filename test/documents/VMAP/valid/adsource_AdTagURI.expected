{
  "version": "1.0",
  "breaks": [
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "source": {
        "id": "1",
        "allowMultipleAds": false,
        "followRedirects": true,
        "dataType": 3,
        "adDataType": 1,
        "adTagURI": "http://7b9df.v.fwmrm.net/ad/1"
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "source": {
        "id": "1",
        "allowMultipleAds": false,
        "followRedirects": true,
        "dataType": 3,
        "adDataType": 2,
        "adTagURI": "http://7b9df.v.fwmrm.net/ad/1"
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "source": {
        "id": "1",
        "allowMultipleAds": false,
        "followRedirects": true,
        "dataType": 3,
        "adDataType": 3,
        "adTagURI": "http://7b9df.v.fwmrm.net/ad/1"
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "source": {
        "id": "1",
        "allowMultipleAds": false,
        "followRedirects": true,
        "dataType": 3,
        "adDataType": 4,
        "adTagURI": "http://7b9df.v.fwmrm.net/ad/1"
      },
      "id": "cue_point-347762684"
    }
  ]
}
